### DISCUSSION TITLE
automatic bitrate

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#4261] [General]
I have a question if I create a 32kbps mount point and give it the default name and create another 128kbps mount point. Will azuracast show the listener's best internet point automatically? that is, azuracast will release the point according to the listener's internet automatically?
As if it were an adaptive bitrate.

**Comment by YosefAdPro** [comment]
No.
It will display them in creative order.

**Reply by YosefAdPro** [reply]
https://features.azuracast.com/suggestions/42104/adaptive-bitrate-streaming
